src/index.js

```diff
@@ -1,10 +1,18 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import { BrowserRouter } from 'react-router-dom';
```

Comment:
Sorry, maybe this is me being picky slash we need to make sure we do a "eslint" on commit as a pre-commit hook. but if you could run "npm run lint" that'd be great.

Reply:
For now, I'll just approve. I'll work on the hook.